apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/index.tsx

```diff
@@ -76,11 +71,15 @@ function nextStepKey(step: StepKey): StepKey {
 const SyncOnboardingManual = () => {
   const { t } = useTranslation();
   const history = useHistory();
-  const [stepKey, setStepKey] = useState<StepKey>(StepKey.Paired);
+  const [stepKey, setStepKey] = useState<StepKey>(StepKey.SoftwareCheck);
 
   const handleSoftwareCheckComplete = useCallback(() => {
     // TODO: put this line instead
     // setStepKey(nextStepKey(StepKey.SoftwareCheck));
+    setStepKey(StepKey.Applications);
```

Comment:
It's better to put the commented line than directly the Application key, this way if the order change we don't need to touch there